src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,53 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static calculator.Calculator.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CalculatorTest
+{
+    // 1. 빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.(예 : “” => 0, null => 0)
+    @Test
```

Comment:
주석으로 작성하셨네요
```suggestion
    @DisplayName("빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.")
    @Test
```
`@DisplayName`을 제공하고 있으니 테스트에 대한 설명을 작성할 수 있어요